specificatie/land.yaml

```diff
@@ -0,0 +1,37 @@
+openapi: 3.0.3
+info:
+  title: Land definities
+  version: 2.0.0
+paths: {}
+components:
+  schemas:
+    AbstractLand:
```

Comment:
paar issues die we moeten bespreken:
1. volgens mij is er geen generieke functionele behoefte aan om te weten dat een land wel bestaat maar onbekend is. Dat is alleen het geval bij verblijfplaats.land, maar die is afgevangen door vertrokkenOnbekendWaarheen. Het type OnbekendLand is dus denk ik niet nodig.
2. Het leveren van een omschrijving uit een landelijke tabel bij een code is geen informatievraag. RvIG mag dus direct zowel code als omschrijving leveren. Een aparte soort GbaLand met alleen code is dus niet nodig.
3. Er is wel punt van het filteren van de onbekendwaarde, dus veld land niet leveren wanneer land.code=0000. Ik denk dat RvIG die wel moet leveren (is in ieder geval nodig bij verblijfplaats.land). De proxy moet die er vervolgens weer uit filteren. Hiervoor hoeft land vanuit RvIG denk ik geen andere component te zijn dan land uit de proxy.